arangod/VocBase/LogicalDataSource.h

```diff
@@ -94,6 +107,12 @@ class LogicalDataSource {
   TRI_vocbase_t& vocbase() const noexcept { return _vocbase; }
 
  protected:
+  // revert a setDeleted() call later. currently only used by LogicalView.
+  // TODO: should be removed
+  void setUndeleted() noexcept {
+    _deleted.store(false, std::memory_order_seq_cst);
```

Comment:
It does not really make sense to make this seq-cst if the other operations on _deleted are all relaxed. It doesn't really hurt, but it might indicate to some reader that there is some synchronization which actually does not exist.

Reply:
+, but also this function should be deleted in future, so why not